java/client/src/org/openqa/selenium/print/PageSize.java

```diff
@@ -0,0 +1,46 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.openqa.selenium.print;
+
+
+public class PageSize {
+
+  private double height;
+  private double width;
+
+  public PageSize() {
+    // Initialize with defaults
+    this.height = 21.59;
```

Comment:
Wat? A comment about where these numbers came from would be really helpful. I'm guessing it's either a4 or letter size paper in cm?

Reply:
Yes in `cm`. Inspiration from https://github.com/SeleniumHQ/selenium/commit/c8bc19e837cd5423be9344d3f69931992115a03a#diff-b7252358e015a773e2d453e794cf8c614fcd330985e0ddf4fe3202392f508e2cR47

Added the comment.